conan/tools/cmake/presets.py

```diff
@@ -6,6 +6,7 @@
 from conan.tools.cmake.layout import get_build_folder_custom_vars
 from conan.tools.cmake.toolchain.blocks import GenericSystemBlock
 from conan.tools.cmake.utils import is_multi_configuration
+import conan.tools.build as build
```

Comment:
Better import directly ``build_jobs``, we are not using scoping in general.

Reply:
would be great if you could do it :)

Reply:
I contributed the test, and also moved the definition to the "common" preset, so it also applies to the ``testPresets``. I understand that we also want to do it, in the same way it affects also the ``CMake.test()`` helper?

Reply:
thanks for the test. very good choice of parallel jobs ;)

this should _not_ be added to the testPresets. I do not have deep enough CMake knowledge to know why, but cmake cannot parse the same jobs parameter in the testPresets.

example invalid presets sections
```
    "buildPresets": [
        {
            "name": "conan-debug",
            "configurePreset": "conan-debug",
            "jobs": 16
        }
    ],
    "testPresets": [
        {
            "name": "conan-debug",
            "configurePreset": "conan-debug",
            "jobs": 16
        }
    ]
```

produces the following error
```
$ cmake --build --target all --preset conan-relwithdebinfo
CMake Error: Could not read presets from <path to root directory>:
Error: @36,21: Invalid extra field "jobs" in Preset
            "jobs": 16
                    ^
```


it builds in parallel with "jobs": 16 in only the buildPresets, and more importantly does not fail to read the presets.

Reply:
I see, yeah, lets remove it!

I'll leave jobs=42 (the answer to everything ;)  only for buildPresets